cmd/tempo/main.go

```diff
@@ -171,11 +179,20 @@ func loadConfig() (*app.Config, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed to parse configFile %s: %w", configFile, err)
 		}
+
+		if configVerify {
```

Comment:
you need to add a line to the top of this method to correctly set the value of `configVerify`. See the lines that set configFileOption and configExpandOption